spec/gogo/x/cash/codec.proto

```diff
@@ -37,13 +37,16 @@ message FeeInfo {
 }
 
 message Configuration {
-  // TODO: add schema uint32 here
+  weave.Metadata metadata = 1;
+  // Owner is present to implement gconf.OwnedConfig interface
+  // This defines the Address that is allowed to update the Configuration object and is
+  // needed to make use of gconf.NewUpdateConfigurationHandler
   bytes owner = 2 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
   bytes collector_address = 3 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
   coin.Coin minimal_fee = 4 [(gogoproto.nullable) = false];
 }
 
 message ConfigurationMsg {
-  // TODO: add schema uint32 here
+  weave.Metadata metadata = 1;
```

Comment:
Now that both `Configuration` model and `ConfigurationMsg` message include schema, they must be registered with migration
- https://github.com/iov-one/weave/blob/master/x/cash/msg.go#L11
- https://github.com/iov-one/weave/blob/master/x/cash/model.go#L13